### ISSUE TITLE
Window object has no attribute 'rfind' --File name argument.

### ISSUE THREAD

**Issue by rakky18** [tyiannak/pyAudioAnalysis#221]
I am trying to use the speakerDiarization() function from audiosegmentation and meanwhile, putting the arguments of the function I encountered this error at the filename argument and was unable to find solution.

I have a fileName stored in a variable containing also the Path of it. It gets no attribute 'rfind' error.
I also tried just putting a plain 'test.wav' into the argument and still it got an error as that.

Traceback (most recent call last): File "C:/Users/Patrick/PycharmProjects/tangina/Thesis/pyqt.py", line 108, in <lambda> 0.2, 0.05, 35)) File "C:/Users/Patrick/PycharmProjects/tangina/Thesis/pyqt.py", line 1245, in speakerDiarization [fs, x] = audioBasicIO.readAudioFile(filename) File "C:\Users\Patrick\PycharmProjects\tangina\venv3\lib\site-packages\pyAudioAnalysis\audioBasicIO.py", line 67, in readAudioFile extension = os.path.splitext(path)[1] File "C:\Users\Patrick\PycharmProjects\tangina\venv3\lib\ntpath.py", line 200, in splitext return genericpath._splitext(p, sep, altsep, extsep) File "C:\Users\Patrick\PycharmProjects\tangina\venv3\lib\genericpath.py", line 99, in _splitext sepIndex = p.rfind(sep) AttributeError: 'Window' object has no attribute 'rfind'``

Has anyone encountered this problem? I searched for solutions in google and non of it works.

**Comment by rakky18** [comment]
I am trying to use the speakerDiarization() function from  audiosegmentation and meanwhile, putting the arguments of the function I encountered this error and was unable to find solution.

I have a fileName stored in a variable containing also the Path of it. It gets no attribute 'rfind' error.
I also tried just putting a plain 'test.wav' into the argument and still it got an error as that.

`Traceback (most recent call last):
  File "C:/Users/Patrick/PycharmProjects/tangina/Thesis/pyqt.py", line 108, in <lambda>
    0.2, 0.05, 35))
  File "C:/Users/Patrick/PycharmProjects/tangina/Thesis/pyqt.py", line 1245, in speakerDiarization
    [fs, x] = audioBasicIO.readAudioFile(filename)
  File "C:\Users\Patrick\PycharmProjects\tangina\venv3\lib\site-packages\pyAudioAnalysis\audioBasicIO.py", line 67, in readAudioFile
    extension = os.path.splitext(path)[1]
  File "C:\Users\Patrick\PycharmProjects\tangina\venv3\lib\ntpath.py", line 200, in splitext
    return genericpath._splitext(p, sep, altsep, extsep)
  File "C:\Users\Patrick\PycharmProjects\tangina\venv3\lib\genericpath.py", line 99, in _splitext
    sepIndex = p.rfind(sep)
AttributeError: 'Window' object has no attribute `'rfind'``

Has anyone encountered this problem? I searched for solutions in google and non of it works.
